### ISSUE TITLE
IfcAlignmentTypeEnum in IFC4

### ISSUE THREAD

**Issue by arishinsemen** [buildingSMART/IDS#332]
In DataTypes.md IFCALIGNMENTTYPEENUM marked "green" IFC4 and IFC4.3.
If look at the documentation ( https://ifc43-docs.standards.buildingsmart.org/IFC/RELEASE/IFC4x3/HTML/lexical/IfcAlignmentTypeEnum.htm) - Changelog: Enum was added in 4.1.
Not in the official standard Ifc4 (IFC4 ADD2 TC1).

![image](https://github.com/user-attachments/assets/860e58a5-7295-43d7-946d-164d91d0b652)

**Comment by arishinsemen** [comment]
Also I couldn't find IFCNULLSTYLEENUM in 2x3 and 4 documentations. In google also :)

**Comment by atomczak** [comment]
you're right, let's update the table after repo housekeeping is finished #335
